### ISSUE TITLE
Feature: Radio configuration over GATT

### ISSUE THREAD

**Issue by ojousima** [ruuvi/ruuvi.firmware.c#89]
e.g. data format, transmission interval, PHYs used, channels used.

**Comment by DG12** [comment]
It seems to me that NFC would be less susceptible to vandalism due to the required proximity and keeping the communications away from BLE would also simplify implementation on both the ruuvi tag and the application to do the setting.

**Comment by ojousima** [comment]
True, there should be some kind of security to unlock DFU updates and configuration.

**Comment by DG12** [comment]
If you're near enough to do NFC then you are near enough to grab the tag and do whatever you want. Simple security to avoid accidentally alterations seems al that is necessary.

**Comment by theBASTI0N** [comment]
You should be able to add passkey to tag just like eddystone tags can be configured with. That would prevent users from changing settings and flashing the device if passkey is not knwon.

**Comment by ojousima** [comment]
Currently NFC scan is required to enable BLE bootloader, I have been thinking about using the unique ID of tag as a passkey. The ID will be exposed to anyone with NFC reader, but I think it would be secure enough as attackers need physical access to tag to configure it.
